internal/ignition/ignition.go

```diff
@@ -342,6 +351,28 @@ func (g *installerGenerator) modifyBMHFile(file *config_31_types.File, bmh *bmh_
 	return nil
 }
 
+func (g *installerGenerator) updateIgnitions() error {
+	masterPath := filepath.Join(g.workDir, "master.ign")
+	caCertFile := g.serviceCACert
+
+	if caCertFile != "" {
+		err := setCACertInIgnition(models.HostRoleMaster, masterPath, g.workDir, caCertFile)
+		if err != nil {
+			return errors.Wrapf(err, "error writing master ignition files")
```

Comment:
These errors could be more specific. Maybe `error adding CA cert to master ignition`? We may end up writing the master ignition file different times, the important part here is we couldn't set up the cert.

Reply:
Done